src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -475,18 +497,15 @@ void Increment(ref long counter)
             // Start N workers, each of which sits in a loop making requests.
             Task.WaitAll(Enumerable.Range(0, concurrentRequests).Select(taskNum => Task.Run(async () =>
             {
+                var clientContext = new ClientContext(client, taskNum: taskNum, seed: seed);
```

Comment:
We don't require named arguments here.

Reply:
While not required, I like to specify named arguments when we have sequences of similarly typed arguments. It's easier to flip your inputs otherwise.